client/functions/index.js

```diff
@@ -0,0 +1,9 @@
+const functions = require('firebase-functions');
```

Comment:
rename firebaseFunctions for consistency with `cloudFunctions` folder and for clarity.